var/spack/repos/builtin/packages/sbml/package.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sbml(CMakePackage):
+    """Library for the Systems Biology Markup Language"""
+
+    maintainers = ['rblake-llnl']
+    homepage = "https://sbml.org"
+
+    def url_for_version(self, version):
+        url = "https://downloads.sourceforge.net/project/sbml/libsbml/{0}/stable/libSBML-{1}-core-plus-packages-src.tar.gz".format(version, version)
+        return url
```

Comment:
Minor OCD: can we move this method below the directives? Asking as that will be consistent with most packages already in the builtin repository.

Reply:
Done!